### PR TITLE
Add reason for removal

### PR DESCRIPTION
**Pull request by DebbieArita**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes https://github.com/opensrp/fhir-resources/issues/150
**Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/assets/configs/app/profiles/household_profile_config.json [hunk 1]
```diff
@@ -386,11 +386,17 @@
...
+                      "data.put('statuses', StringUtils:join([data.get('getChildStatus'), data.get('getSickChildStatus'), data.get('getPregnancyStatus'), data.get('getFPStatus'), data.get('getPNCStatus'), data.get('getMentalHeathStatus'), data.get('getHIVStatus'),data.get('getCMNTDStatus'), data.get('getTBStatus'), data.get('removalReason')], ', ').replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceFirst(\"^, \", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\").replaceAll(\", $\", \"\"))"
```
[the remark this answers is not in the document]
Comment (DebbieArita) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json:399]: That's actually not necessary... removing that...

[the remark this answers is not in the document]
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json]: They might be necessary unless we added code to remove trailing spaces and commas.

[the remark this answers is not in the document]
Comment (DebbieArita) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json]: @dubdabasoduba... There was an extra one I added by mistake on that line.... that's what I corrected...

File: android/quest/src/main/assets/configs/app/profiles/household_profile_config.json [hunk 2]
```diff
@@ -386,11 +386,17 @@
                       "data.put('getSickChildStatus', service.mapResourcesToLabeledCSV(Condition, \"Condition.code.coding.code = '275142008' and Condition.clinicalStatus.coding.code = 'active'\", 'Sick Child'))"
                     ]
                   },
+                  {
+                    "name": "removalReason",
+                    "condition": "true",
+                    "actions": [
+                      "data.put('removalReason', fhirPath.extractValue(Patient, Patient.active) == 'false' ? fhirPath.extractValue(Observation, 'Observation.code.text') : '')"                    ]
```
[the remark this answers is not in the document]
Comment (ellykits) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json:393]: The profile configuration resources have not been updated to include Observation resource

File: android/quest/src/main/assets/configs/app/profiles/household_profile_config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json]: @ellykits can the work done to remove the trailing comma handle this?

File: android/quest/src/main/assets/configs/app/profiles/household_profile_config.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/profiles/household_profile_config.json:397]: @DebbieArita is this supposed to be `!=` or `==`. Also, you might not need it based on this https://github.com/opensrp/fhir-resources/blob/main/ecbis_liberia/app_configurations/ecbis_cha_preview/profiles/default_profile_config.json#L53-L59c

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**